docs/docs/cli.md

```diff
@@ -445,7 +445,7 @@ poetry export -f requirements.txt > requirements.txt
 * `--extras (-E)`: Extra sets of dependencies to include.
 * `--without-hashes`: Exclude hashes from the exported file.
 * `--with-credentials`: Include credentials for extra indices.
-
+* `--allow-prereleases`: Install a pre-release version if present for the library. Example: poetry add --allow-prereleases=true
```

Comment:
Hello @Nirvi1,

thanks a lot for your contribution!

Your suggested changed should be belong to the `add` command and not to `export`, shouldn't it?

fin swimmer

Reply:
Thanks for this! Just changed the doc line to add, Can you review again?